scripts/Dockerfile-build

```diff
@@ -15,5 +15,18 @@ ENV NAME=${NAME}
 
 RUN CGO_ENABLED=1 GOOS=linux go build -a -installsuffix cgo -ldflags "-extldflags \"-static\" -s -w -X main.GitCommit=$GIT_COMMIT -X main.BuildVersion=$VERSION -X main.BuildType=$NAME" -o owncast .
 
+# Create the image by copying the result of the build into a new alpine image
+FROM alpine
+RUN apk add --no-cache ffmpeg ffmpeg-libs
+
+# Certs
+RUN apk update && apk add ca-certificates && update-ca-certificates
```

Comment:
I’ve never needed to run the update.  Also I’d combine this install into previous run so you keep the number of layers small

Reply:
Let me know if this is along the lines you were thinking of:

https://github.com/owncast/owncast/pull/188/commits/e41104c0c8d78c35f0d0fcb2e9ec110c629da07c#diff-26db72de85beb4285d47e4cc9f631449R20